demo/server/config-dev.js

```diff
@@ -12,14 +12,16 @@ var config = {
   "security": {
     "adminRole": "admin",
     "userRole": "user",
+    "allowedRoles": ["adminRole"],
```

Comment:
-1. Roles can be specified directly in the places we need them. 
Why we need this in the server?

Reply:
would you agree to rename this to say `portalAllowedRoles`? I think we should avoid adding this to the client side.